examples/look/requirements/test

```diff
@@ -1,3 +1,5 @@
+gunicorn==19.9.0
```

Comment:
Note that our recommended way to run the _look_ example with `gunicorn` does not work on Gunicorn 20.0.0.

However, if I interpret https://github.com/benoitc/gunicorn/issues/2175 correctly, a fix may be already in `master` or under way.

Reply:
For what it's worth, I generally import and use gunicorn directly in my apps when wrapping gunicorn. We might investigate doing that here as well.

Reply:
Yeah, that's generally a good approach. I have adopted it at work for some development servers (we use Nginx+uWSGI on prod though) as it's easy to toss in a static route, inject some dependencies or w/e needed for the dev environment.

However, I'm not sure that's a good approach here in this "look" tutorial as we recommend running gunicorn [like that](https://falcon.readthedocs.io/en/stable/user/tutorial.html#testing-your-application) and [like that](https://falcon.readthedocs.io/en/stable/user/tutorial.html#refactoring-for-testability). We could of course improve that, but IMHO running gunicorn in the command line is arguably easier for new users. Plus, it's much easier to swap out command-line unicorn to something else, for instance, waitress for Windows folks etc.